K2Field.K2NE.ServiceBroker/ServiceObjectBase.cs

```diff
@@ -56,7 +56,7 @@ public string LDAPPaths
         {
             get
             {
-                return this.ServiceBroker.Service.ServiceConfiguration[Constants.ConfigurationProperties.LDAPPaths].ToString();
+                return this.ServiceBroker.Service.ServiceConfiguration[Constants.ConfigurationProperties.LDAPPaths].ToStringOrEmpty();
```

Comment:
At the moment, If the value of this property (and other properties) is empty - then AD service objects simply return nothing. So i think there is no need to throw an exception. Or there should be a logic to enable\disable AD feature and if it is enabled - then check values for emptiness.